index.html

```diff
@@ -3,34 +3,40 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="styles.css">
   </head>
-  <body>
-    <div>
 
 
+  <body class="container">
+
+    <div>
+      <section class="first_block">
       <nav>
-        <ul>
-          <li>
+        <ul class="list">
+          <li class="startly">
```

Comment:
I wonder if the li elements should have been id's rather than classes as they're targeting one element rather than multiple elements.